### PR TITLE
Add install instructions

### PR DESCRIPTION
**Pull request by froboy**
Based on a few Slack conversations, here's a first pass at converting @podarok's video tutorial into text.

### CODE REVIEW

**Review by jamieshiz — CHANGES_REQUESTED** [review]
Maybe include troubleshooting tips....
I got this error when trying to enable the modules via drush. Solution was to execute from docroot folder
```
Command pm-enable needs a higher bootstrap level to run - you will need to invoke drush from a more functional Drupal environment to run this command.                                            [error]
The drush command 'en openy_gated_content openy_gc_storage openy_gc_auth_example' could not be executed.     
```

Also maybe an explanation on which modules to enable and why. For example, I will be using openy_gc_auth_custom module for authenticating with CCC

**Review by anpolimus — APPROVED** [review]

**Comment by anpolimus** [comment]
@jamieshiz Thank you for your comment, but it is a common thing to use drush from project docroot.
>Also maybe an explanation on which modules to enable and why. For example, I will be using openy_gc_auth_custom module for authenticating with CCC

We will write it for the Auth subsystem here: https://github.com/fivejars/openy_gated_content/tree/master/modules/openy_gc_auth

File: README.md [hunk 1]
```diff
@@ -2,10 +2,34 @@
...
+     - Check "Virtual Y Base", "Open Y Virtual YMCA Storage", and "Open Y
+      Virtual YMCA Auth Example".
+      - Install and say "yes" to add all required dependencies.
+   - Via drush: `drush en openy_gated_content openy_gc_storage
+    openy_gc_auth_example -y`
+- Add at least one term in these vocabularies via **Structure** > **Taxonomies**
+  - Virtual Y Category
+  - Virtual Y Equipment
+  - Virtual Y Level
+- Create a **Landing Page** and add the **Virtual Y Content** paragraph to the
```
Comment (hamrant) [inline README.md:25]: After the user segmentation merge we also need a page with a login paragraph and steps for a setup these pages URLs in settings.

cc @froboy @anpolimus

Comment (anpolimus) [reply README.md:25]: @hamrant I'll add it in my next PR.
Lets merge it for now

**Pull request merged**